src/Shared/HandyControlDemo_Shared/Properties/Langs/Lang.pl.resx

```diff
@@ -358,7 +355,7 @@
     <value>Czy uruchomić ponownie aby zmienić język?</value>
   </data>
   <data name="About" xml:space="preserve">
-    <value>O...</value>
+    <value>Około</value>
```

Comment:
"Około" in Polish is most often used as "around" and not "about".
Most programs and companies use "O something" when they mean "About something".
Example from Firefox:
![obraz](https://user-images.githubusercontent.com/23077337/105163156-d6baba80-5b13-11eb-9281-d493bedd5924.png)
![obraz](https://user-images.githubusercontent.com/23077337/105163168-dae6d800-5b13-11eb-873f-ebda382d0906.png)

Alternatively "about" alone could be translated as "Informacje" ("Informations"), as translated in VS Code.
![obraz](https://user-images.githubusercontent.com/23077337/105163415-2d27f900-5b14-11eb-9258-caa9a8028310.png)
![obraz](https://user-images.githubusercontent.com/23077337/105163479-3e710580-5b14-11eb-8b95-8c898c1ed5cb.png)

Reply:
should it be O... or O?

Reply:
After a longer consideration, I came to a conclusion, that "Informacje" would look better in there.
Both "O" and "O..." would be a little too short

Reply:
update in 1ae275a2